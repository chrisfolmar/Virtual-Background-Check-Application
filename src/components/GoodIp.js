import React, { Component } from 'react'

/**
 * A component to hold the data when a searched IP is not found to be blacklisted
 */
 class GoodIp extends Component {

	/**
     * render
     * @return {ReactElement} markup
     */
     render(){
     	const address = this.props.address;

     	var greenStyle = {
     		color: '#06B400'
     	};

     	return(
     		<div className="features-icons-item mx-auto mb-5 mb-lg-0 mb-lg-3">
     			<h4>{address}</h4>
     			<h5 style={greenStyle}>Not Found!</h5>
     			<p className="mb-0">The IP was not found in the database of blacklists!</p>
     		</div>
     		);
     }
 }

 export default GoodIp;