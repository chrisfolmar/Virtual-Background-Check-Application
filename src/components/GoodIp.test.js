import React from 'react';
import ReactDOM from 'react-dom';
import GoodIp from './GoodIp';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<GoodIp />, div);
  ReactDOM.unmountComponentAtNode(div);
});