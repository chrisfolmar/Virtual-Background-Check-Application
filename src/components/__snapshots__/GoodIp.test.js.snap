// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoodIp full snapshot rendering 1`] = `
<GoodIp>
  <div
    className="features-icons-item mx-auto mb-5 mb-lg-0 mb-lg-3"
  >
    <h4 />
    <div
      className="features-icons-icon d-flex"
    >
      <i
        className="fa fa-check-circle-o m-auto"
        style={
          Object {
            "color": "#06B400",
          }
        }
      />
    </div>
    <h5
      style={
        Object {
          "color": "#06B400",
        }
      }
    >
      Not Found!
    </h5>
    <p
      className="mb-0"
    >
      The IP was not found in the database of blacklists!
    </p>
  </div>
</GoodIp>
`;
